lib/expression.js

```diff
@@ -233,6 +234,12 @@ CronExpression._parseField = function _parseField (field, value, constraints) {
           max = Math.max.apply(Math, stack);
         }
       } else { // Scalar value
+        //TODO: handle the cases when there is a range and L, or list of dates and L
+        if(field === "dayOfMonth" && result === "L") {
```

Comment:
```suggestion
        if(field === 'dayOfMonth' && result === 'L') {
```
No linting rules that enforce single-quotes here but they are definitely preferred.